README.md

```diff
@@ -231,13 +247,28 @@ You can run some unit tests with:
 npm test
 ```
 
-The supervisor runs on Node v10.19.0, so using that specific
+The supervisor runs on Node v12.16.2, so using that specific
 version will ensure tests run in the same environment as
 production.
 
 Alternatively, tests will be run when building the image,
 which ensures that the environment is exactly the same.
 
+## Troubleshooting
+
+Make sure you are running atleast:
+
+```sh
+node -v       # >= 12.16.2
+npm -v        # >= 6.14.0
+docker -v     # >= 19.03.0
```

Comment:
There is no need to specify docker and git

Reply:
I saw logs from Husky saying I need Git >= 2.13 when installing dependencies. I added docker since this was in the troubleshooting section so it was helpful to add versions that are confirmed to be working.

```sh
> husky@4.2.3 install /home/mig/Projects/balena.io/balena-supervisor/node_modules/husky
> node husky install

husky > Setting up git hooks
Husky requires Git >=2.13.0. Got v2.11.0.
husky > Failed to install
```

Reply:
Fair enough, although that docker version for sure isn't required, we would work with *much* earlier versions. I'd prefer to simply specify that docker is required, or to add the actual version of docker that would support things, but that's too much of a yakshave IMO.

Reply:
Removed docker version since it would just complicate this PR to find out the oldest version needed.